src/utils_social.js

```diff
@@ -288,6 +312,13 @@ try {
      * https://www.instagram.com/cristiano/followers
      * ```
      *
+     * It also does NOT match the following URLs:
+     * ```
+     * https://www.instagram.com/explore/
+     * https://www.instagram.com/_n/
```

Comment:
what are `_n` and `_u` URLs? never seen this

Reply:
I also don't know, I just noticed there was a TODO for excluding these